packages/trader/src/AppV2/Containers/Chart/trade-chart.tsx

```diff
@@ -89,19 +85,6 @@ const TradeChart = observer(() => {
 
     const { current_spot, current_spot_time } = accumulator_barriers_data || {};
 
-    const bottomWidgets = React.useCallback(({ digits, tick }: TBottomWidgetsParams) => {
-        return (
-            <BottomWidgetsMobile digits={digits} tick={tick} setTickData={setTickData} setDigitStats={setDigitStats} />
-        );
-    }, []);
-
-    const chartKey = React.useMemo(() => {
```

Comment:
Removed this forced chart update. Purpose of new `chartKey` was to fix the bug from DTRA-2029, but actually it creates issue with loader. 
So, I removed this logic (and one of the loader issue was fixed) and fix the bug from DTRA-2029 in another way.